examples/solid/simple/.eslintrc

```diff
@@ -6,6 +6,7 @@
   },
   "rules": {
     "react/react-in-jsx-scope": "off",
-    "jsx-a11y/accessible-emoji": "off"
+    "jsx-a11y/accessible-emoji": "off",
+    "@tanstack/query/prefer-query-object-syntax": "off"
```

Comment:
good point - we should probably remove that rule for v5, because TypeScript will enforce the correct usage for us.
- https://github.com/TanStack/query/issues/4739

Reply:
Heh! I wouldn't have necessarily removed the lint rule but since the only argument now is a function. The lint rule complains because it thinks we're passing it a `queryFn`